pkg/config/config_test.go

```diff
@@ -466,6 +466,15 @@ func TestAddAgentVersionToDomain(t *testing.T) {
 	require.Nil(t, err)
 	assert.Equal(t, "https://"+getDomainPrefix("flare")+".datadoghq.eu", newURL)
 
+	// Gov
+	newURL, err = AddAgentVersionToDomain("https://app.ddog-gov.com", "app")
+	require.Nil(t, err)
```

Comment:
You may know (and you're probably mimicking the rest of the test) but just in case: there is a `require.NoError`

Reply:
Yeah, that was just copy-pasted from elsewhere in the test.